src/mon/Monitor.cc

```diff
@@ -2652,6 +2664,13 @@ void Monitor::handle_command(MonOpRequestRef op)
   boost::scoped_ptr<Formatter> f(Formatter::create(format));
 
   get_str_vec(prefix, fullcmd);
+
+  // before get fullcmd[0], verify string to vector success.
+  if (fullcmd.empty()) {
```

Comment:
Hmm, I think there is, for a prefix only consist by delim characters, say ";;;;"
see get_str_vec